### PR TITLE
Feature milestone 1

### PR DESCRIPTION
**Pull request by nurislam-ab**
This PR has the following:
- [x] React setup
- [x] Linters setup
- [x] Heroku deploy

### CODE REVIEW

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup ESLint
+        run: |
+          npm install --save-dev eslint@6.8.x eslint-config-airbnb-base@14.1.x eslint-plugin-import@2.20.x
+          [ -f .eslintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/javascript/.eslintrc.json
+      - name: ESLint Report
+        run: npx eslint .
```
Comment (Torres-ssf) [inline .github/workflows/linters.yml:22]: - [ ] This file was updated 2 weeks ago. We need to use to the most recent one. You can find it [here](https://github.com/microverseinc/linters-config/blob/master/react-redux/.github/workflows/linters.yml):

**Review by Torres-ssf — CHANGES_REQUESTED** [review]
### Changes required :muscle: 
Hi @nurislam-ab :hand: 
Awesome job in this milestone :clap: 
But we need to make some changes before moving on.

- [x] We need to improve the README file by adding relevant information about your project. This requirement is listed on Microverse General Requirements. You can use this template [here](https://github.com/microverseinc/readme-template). Remember to also inform the link from the live version of your application.
Screenshot from 2020-10-12 15-00-18

- [x] Awesome job cleaning your project and deleting the unnecessary files. However, there was no need to delete index.js and app.js. Instead of that what can be done in the setup stages is to make the App component to return only a simple message (like `Hello World`) on an `h1` or `p` element.

Feel free to message me on slack if you have any questions :smile: 
Happy Coding :computer: 
**Sergio Torres**

**Review by mikenath223 — APPROVED** [review]
### Hi @nurislam-ab 👋

**Nice work so far on this. 👍**

### STATUS: REQUESTED CHANGES 👷‍♂️

- [x] - OPTIONAL - Please consider changing the files extension of the files in the directory of `src/components` from `jsx` to simply `js`.

- [ ] - OPTIONAL - Is there a reason why your Github actions run twice for each lint action? Kindly consider fixing it.

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

HAPPY CODING👨‍💻

**Comment by nurislam-ab** [comment]
> * [ ]  - OPTIONAL - Is there a reason why your Github actions run twice for each lint action? Kindly consider fixing it.

Tried to find the reason but was not able. 
Removed linters and did it again but still, it runs twice

**Pull request merged**